landlab/components/uniform_precip/generate_uniform_precip.py

```diff
@@ -13,7 +13,8 @@
 
 import random
 import numpy as np
-from landlab import Component
+from landlab import Component, ModelGrid
+from six import next
 
 
```

Comment:
@SiccarPoint Sorry, was backpacking (and now en route to the Boundary Waters) so my response time is somewhat delayed. 

You are right, I can't think of a component where this is implemented. I''ve looked around and found mostly units in meters and seconds. 

I suspect that units were a large topic of discussion before I joined the team, but maybe its worth revisiting in a future call since as far as I can tell landlab is agnostic, but most components list units in meters, seconds, etc.